### PR TITLE
Updates JS examples to protobufjs + matrix-protos

### PR DESCRIPTION
**Pull request by maciekrb**
- Node JS examples now use matrix-protos module
- Node JS examples updated to protobufjs 6.x

### CODE REVIEW

**Review by arhuaco — COMMENTED** [review]
I always wanted to convert to the the 6.X version.
Leave this PR as an example somewhere.

**Review by eighteyes — APPROVED** [review]
Works good for me

File: src/js_test/test_humidity.js [hunk 1]
```diff
@@ -39,25 +35,25 @@ errorSocket.on('message', function(error_message) {
...
-driverConfigProto.timeout_after_last_ping = 6.0
-var hum_params_msg = new matrixMalosBuilder.HumidityParams
-// Real current temperature [Celsius] for calibration 
-hum_params_msg.current_temperature = 23
-driverConfigProto.set_humidity(hum_params_msg)
+var config = matrix_io.malos.v1.driver.DriverConfig.create({
+  delayBetweenUpdates: 2.0,  // 2 seconds between updates
+  timeoutAfterLastPing: 6.0, // Stop sending updates 6 seconds after pings.
+  humidity: matrix_io.malos.v1.sense.HumidityParams.create({
+    currentTemperature: 23   // Real current temperature [Celsius] for calibration
```
Comment (arhuaco) [inline src/js_test/test_humidity.js:42]: Perhaps we should put this on top where it is easier to spot?

**Review by arhuaco — COMMENTED** [review]
Looks good overall. Two comments.

1) You should get review from the author of each file for the respective approval.
2) And make sure that the console.log is printing something meaningful. Perhaps this changed with the upgrade.

**Review by hpsaturn — APPROVED** [review]

**Review by eljuguetero — APPROVED** [review]

File: src/js_test/test_humidity.js (hunk 1 above)
Comment (maciekrb) [reply src/js_test/test_humidity.js:42]: 1. You're right.  I'm pinging @eljuguetero to approve on this as well. For the other files, I think we are covered with @hpsaturn and yourself. 
2. I checked the outputs of all tests have you noticed something specially ugly ? 

Regarding the humidity test, I'm following the same logic as other examples : 
  - error socket connection / binding
  - configuration socket connection and messaging
  - update socket connection / binding
  - ping socket connection / messaging 

You think it makes sense to move the config logic to the top in all of the samples ?

**Review by arhuaco — APPROVED** [review]

File: src/js_test/test_humidity.js (hunk 1 above)
Comment (arhuaco) [reply src/js_test/test_humidity.js:42]: Mmm. Better to leave things as they are. We need errors first so we get them if the config goes wrong.

**Pull request merged**